lib/common/optional/types.go

```diff
@@ -166,6 +166,27 @@ func (x *Bytes) Decode(r io.Reader) (*Bytes, error) {
 	return x, nil
 }
 
+// DecodeBytes return an optional Bytes from scale encoded data
+func (x *Bytes) DecodeBytes(data []byte) (*Bytes, error) {
```

Comment:
please add unit test for this, test coverage is low

Reply:
Done